test/select.spec.js

```diff
@@ -2894,6 +2894,28 @@ describe('ui-select tests', function () {
       expect(scope.pasteFunc).toHaveBeenCalledWith('tag1');
     });
 
+    it('should broadcast selected items when paste function returns pasted items', function() {
+      scope.pasteFunc = function(string) { return [string] };
```

Comment:
nit: Calling the arg here `string` caught me off guard. I'd prob just call this `str`, but not a huge deal.